pkg/environment/utils.go

```diff
@@ -75,6 +76,19 @@ func GetResolvedChannel(channel string, inputChan, outputChan []brokers.ChannelB
 	).InvalidChannel()
 }
 
+// GetRouteData returns the data of a resolved route
+func GetRouteData(route string) (*meta.RouteConnection, error) {
+	value, exists := os.LookupEnv(route + "_ROUTE")
+	if !exists {
+		return nil, ierrors.New("invalid route: %s", route).BadRequest()
```

Comment:
I think it would be better to change the error message to something like: "route %v not found in enviroment variables"